taca/illumina/HiSeqX_Runs.py

```diff
@@ -65,7 +65,7 @@ def demultiplex_run(self):
         if not os.path.exists(samplesheet_dest):
             try:
                 with open(samplesheet_dest, 'wb') as fcd:
-                    fcd.write(_generate_clean_samplesheet(ssparser, fields_to_remove=['index2'], rename_samples=True, rename_qPCR_suffix = True, fields_qPCR=['SampleName']))
+                    fcd.write(_generate_clean_samplesheet(ssparser, fields_to_remove=['index2'], rename_samples=True, rename_qPCR_suffix = True, fields_qPCR=[ssparser.dfield_snm]))
```

Comment:
I added only three (crucial maybe) variables in this [PR](https://github.com/SciLifeLab/flowcell_parser/pull/15/files). Do you want to add variables for "one worded" headers as well ?